decidim-elections/app/views/decidim/elections/admin/steps/_tally_ended.html.erb

```diff
@@ -0,0 +1,9 @@
+<div class="card">
+  <div class="card-divider">
+    <h2 class="card-title"><%= t(".title") %></h2>
+  </div>
+
+  <div class="card-section">
+    Tally ended
```

Comment:
I18n 👀

Reply:
This text was just a placeholder for the page content. @slickepinne is adding content to the page in [this PR](#7276)